spec/graphql/mutations/invoices/generate_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Mutations::Invoices::Download, type: :graphql do
```

Comment:
RSpec/FilePath: Spec path should end with `mutations/invoices/download*_spec.rb`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)